modules/layers/src/mode-handlers/snappable-handler.js

```diff
@@ -103,12 +103,42 @@ export class SnappableHandler extends ModeHandler {
     }
   }
 
+  // If pickFromOtherLayerIds is present in modeConfig, this method will return the features
+  // from the specified layers. Additionally, if appendPicksFromOtherLayers is set to true
+  // in modeConfig, the features from other layers will be appended to the features in
+  // this._handler. Otherwise, this method will simply return the features from this._handler.
+  _getFeaturesFromRelevantLayer(): any[] {
+    let features = [];
+    const { pickFromOtherLayerIds, appendPicksFromOtherLayers } = this._modeConfig || {};
+
+    if (pickFromOtherLayerIds) {
+      const otherLayersToPickFrom = this._context.layerManager.layers.filter(
+        layer => pickFromOtherLayerIds && pickFromOtherLayerIds.includes(layer.id)
+      );
+
+      features = otherLayersToPickFrom
+        .map(otherLayer => otherLayer.props.data)
+        .reduce((a, b) => [...a, ...b], []);
+
+      if (!appendPicksFromOtherLayers) {
```

Comment:
Why do you need to check both?

Reply:
Are you saying that we don't need the `appendPicksFromOtherLayers` modeConfig property, such that if `pickFromOtherLayerIds` property is populated, automatically pick from those layers in addition to the current layer?